src/stores/routePathStore.ts

```diff
@@ -14,23 +14,30 @@ export enum AddLinkDirection {
     AfterNode,
 }
 
+export enum RoutePathViewTab {
+    Info,
+    List,
+}
+
 export class RoutePathStore {
     @observable private _isCreating: boolean;
     @observable private _routePath: IRoutePath|null;
     @observable private _hasUnsavedModifications: boolean;
     @observable private _isGeometryValid: boolean;
     @observable private _neighborRoutePathLinks: IRoutePathLink[];
-    @observable private _highlightedNodes: string[];
-    @observable private _highlightedLinks: string[];
+    @observable private _highlightedObjects: string[];
```

Comment:
Why is this an array? When can multiple items be highlighted?

Reply:
When we are highlighting nodes from where you can continue building routepath, then there are many nodes highlighted

Reply:
hmm, but then this variable is not used. Changed this now to be singular